src/main/java/com/google/sps/data/Comment.java

```diff
@@ -13,11 +13,10 @@
   private final String author;
   private String messageContent;
   private Date timestamp;
-  private Long id;
-  private Long total;
+  private long id;
+  private Long negative;
```

Comment:
why did you lowercase long id but not long positive/negative?

Reply:
Yeah, do you purposefully use the Long object?